src/routes/user/userparentalcontrol.tsx

```diff
@@ -168,7 +168,7 @@ const UserParentalControl: FunctionComponent = () => {
         populateRatings(allParentalRatings);
         let ratingValue = '';
 
-        if (user.Policy.MaxParentalRating) {
+        if (user.Policy.MaxParentalRating !== null) {
```

Comment:
Shouldn't we also check for `undefined`?
```suggestion
        if (user.Policy.MaxParentalRating != null) {
```